Gopkg.toml

```diff
@@ -35,23 +35,23 @@
 
 [[constraint]]
   name = "k8s.io/apiextensions-apiserver"
-  version = "kubernetes-1.13.3"
+  version = "kubernetes-1.13.11"
 
 [[override]]
   name = "k8s.io/api"
-  version = "kubernetes-1.13.3"
+  version = "kubernetes-1.13.11"
 
 [[override]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.13.3"
+  version = "kubernetes-1.13.11"
 
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "=v10.0.0"
+  version = "v10.0.0"
```

Comment:
Did you remove `=` here and below on purpose? Without it dep may choose the latest patch release which may potentially break things (I know patch releases should be compatible but in practice this is not always the case).